web/src/main/java/org/cbioportal/genome_nexus/web/EnsemblController.java

```diff
@@ -87,7 +92,11 @@ public EnsemblController(EnsemblService ensemblService)
 
         for (String proteinId : proteinIds)
         {
-            transcripts.addAll(this.ensemblService.getEnsemblTranscriptsByProteinId(proteinId));
+            try {
+                transcripts.addAll(this.ensemblService.getEnsemblTranscriptsByProteinId(proteinId));
+            } catch (EnsemblTranscriptNotFoundException e) {
+                e.printStackTrace();
```

Comment:
This doesn't seem right. You should probably have another service method that takes a list and returns a list, and wouldn't throw this exception.